core/src/test/java/com/coderanch/blackjack/HandTest.java

```diff
@@ -0,0 +1,173 @@
+import com.coderanch.blackjack.Card.Suit;
+
+import org.junit.experimental.theories.DataPoints;
+import org.junit.experimental.theories.Theories;
+import org.junit.experimental.theories.Theory;
+import org.junit.runner.RunWith;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertThrows;
+
+
+/**
+ * Tests the {@link Hand} class.
+ */
+@RunWith(Theories.class)
+public final class HandTest {
+
+    /**
+     * Hands to test scoring.
+     */
+    @DataPoints
+    public static final List<HandTestArgument> HANDS =
+            List.of(
+                    new HandTestArgument(
+                            List.of(new Card(Rank.ACE, Suit.CLUBS),
+                                    new Card(Rank.ACE, Suit.CLUBS),
+                                    new Card(Rank.KING, Suit.CLUBS),
+                                    new Card(Rank.EIGHT, Suit.CLUBS),
+                                    new Card(Rank.ACE, Suit.CLUBS)),
+                            21
+                    ),
+
+                    new HandTestArgument(
+                            List.of(new Card(Rank.QUEEN, Suit.CLUBS),
+                                    new Card(Rank.EIGHT, Suit.CLUBS)),
+                            18
+                    ),
+
+                    new HandTestArgument(
+                            List.of(new Card(Rank.QUEEN, Suit.CLUBS),
+                                    new Card(Rank.ACE, Suit.CLUBS)),
+                            21
+                    ),
+
+                    new HandTestArgument(
+                            List.of(new Card(Rank.KING, Suit.CLUBS),
+                                    new Card(Rank.KING, Suit.CLUBS),
+                                    new Card(Rank.KING, Suit.CLUBS),
+                                    new Card(Rank.KING, Suit.CLUBS),
+                                    new Card(Rank.KING, Suit.CLUBS)),
+                            0
+                    ),
+
+                    new HandTestArgument(
+                            List.of(new Card(Rank.ACE, Suit.CLUBS),
+                                    new Card(Rank.ACE, Suit.CLUBS),
+                                    new Card(Rank.ACE, Suit.CLUBS),
+                                    new Card(Rank.KING, Suit.CLUBS),
+                                    new Card(Rank.KING, Suit.CLUBS)),
+                            0
+                    ),
+
+                    new HandTestArgument(
+                            List.of(new Card(Rank.ACE, Suit.CLUBS),
+                                    new Card(Rank.ACE, Suit.CLUBS),
+                                    new Card(Rank.NINE, Suit.CLUBS)),
+                            21
+                    )
+            );
+
+    /**
+     * Cards to test.
+     */
+    @DataPoints("objects")
+    public static final Set<Card> CARDS = Cards.getStandardDeck();
```

Comment:
The `"objects"` data point is meant to provide instances of the class under test. For `HandTest`, that would be a set of hands, not a set of cards.

Reply:
I see. So I should remove it or refactor the tests?

Reply:
I admit I still have to look over the tests a bit more, but you probably don't have to refactor it for now. Just remove the `"objects"` identifier from the data point.